objects.go

```diff
@@ -63,11 +97,130 @@ type Network struct {
 	Ea          EA     `json:"extattrs,omitempty"`
 }
 
+type Members []Member
+
+type ServiceStatus struct {
+	Desciption string `json:"description,omitempty"`
+	Service    string `json:"service,omitempty"`
+	Status     string `json:"status,omitempty"`
+}
+
+type LanHaPortSetting struct {
+	HaPortSetting  map[string]interface{} `json:"ha_port_setting,omitempty"`
+	LanPortSetting map[string]bool        `json:"lan_port_setting,omitempty"`
+}
+
+type NodeInfo struct {
+	HaStatus             string                 `json:"ha_status,omitempty"`
+	HwId                 string                 `json:"hwid,omitempty"`
+	HwModel              string                 `json:"hwmodel,omitempty"`
+	HwPlatform           string                 `json:"hwplatform,omitempty"`
+	HwType               string                 `json:"hwtype,omitempty"`
+	Lan2PhysicalSetting  map[string]bool        `json:"lan2_physical_setting,omitempty"`
```

Comment:
Why is this `map[string]bool`?

According to the WAPI doc, it should be an array of struct like this:

> physicalportsetting : Physical Port Settings.
This structure contains settings for an appliance physical port.
auto_port_setting_enabled
auto_port_setting_enabled

>Enable or disalbe the auto port setting.
Type
Bool.
Create
The default value is undefined.
duplex
duplex

>The port duplex; if speed is 1000, duplex must be FULL.
Type
String.
Valid values are:
FULL
HALF
Create
The default value is undefined.
speed
speed

>The port speed; if speed is 1000, duplex is FULL.
Type
String.
Valid values are:
10
100
1000
Create
The default value is undefined.

Reply:
output contains of the below form:

for instance:

"lan_ha_port_setting": {
                    "ha_port_setting": {
                        "auto_port_setting_enabled": true, 
                        "speed": "10"
                    }, 
                    "lan_port_setting": {
                        "auto_port_setting_enabled": true
                    }
                }

so its not array. hence used maps.

Reply:
It should be a struct, i.e.

`Lan2PhysicalSetting     LanPhysicalSetting`

where,

` type LanPhysicalSetting struct {`
`       AutoPortEnabled       bool`
`       duplex                       string`
etc...

I said "array of struct" because I thought there's multiple of these.

Reply:
Done